packages/zosfiles/src/api/methods/create/Create.ts

```diff
@@ -134,6 +134,25 @@ export class Create {
         }
     }
 
+    public static async dataSetLike(session: AbstractSession, dataSetName: string, likeDataSetName: string): Promise<IZosFilesResponse> {
+        // Required
+        ImperativeExpect.toNotBeNullOrUndefined(dataSetName, ZosFilesMessages.missingDatasetName.message);
+        ImperativeExpect.toNotBeNullOrUndefined(likeDataSetName, ZosFilesMessages.missingDatasetLikeName.message);
+
+        try {
+            const endpoint: string = ZosFilesConstants.RESOURCE + ZosFilesConstants.RES_DS_FILES + "/" + dataSetName;
+
+            const data = await ZosmfRestClient.postExpectString(session, endpoint, [], { like: likeDataSetName });
```

Comment:
Could you stringify the `{like: likeDataSetName}` object before passing it to the expectString function?
This will prevent issues caused by what we believe is an old issue, (or maybe it was introduced not long ago?)